README.md

```diff
@@ -1,4 +1,35 @@
 # xfero
-File Transfer Orchestration software
+A practical File Transfer Orchestration solution for business
 
-This needs filling in
+## Why xfero?
+**THE MISSING LINK IN BUSINESS FILE TRANSFER SERVICES**
+
+Xfero is the complete orchestration framework for business file transfer services with a mission to simplify complex file transfer integration while giving business users the ability to manage configuration rather than relying on techies.
```

Comment:
So, the pedant in me has never liked starting a sentence with a lower case letter but it is something I've got used to, so my view isn't based on grammatical correctness but, when I've been reading the wiki pages I have found 'Xfero' more eye catching when it occurs in the middle of a sentence, e.g. "...removes the complexities of configuration via Xfero Navigator"